src/Components/Admin/Admins/Form/index.jsx

```diff
@@ -97,7 +97,7 @@ const AdminsForm = ({ history }) => {
   return (
     <div>
       <form className={styles.form} onSubmit={handleSubmit(onSubmit, onInvalid)}>
-        <div className={styles.formContainer}>
+        <div className={styles.formContainer} testId="admin-form">
```

Comment:
following the style we use for name the other tests id this should be admin-profile-form